addons/knobs/README.md

```diff
@@ -217,6 +217,28 @@ const value = select(label, options, defaultValue);
 
 > You can also provide options as an array like this: `['red', 'blue', 'yellow']`
 
+### selectV2
+
+Future replacement for `select`. The value from the select now uses the values from the `options` object.
+
+```js
+import { selectV2 } from '@storybook/addon-knobs';
+
+const label = 'Colors';
+const options = {
+  Red: 'red',
+  Blue: 'blue',
+  Yellow: 'yellow',
+  Rainbow: ['red', 'orange', 'etc'],
+  Black: null,
+};
+const defaultValue = 'Red';
+
+const value = select(label, options, defaultValue);
```

Comment:
effectively, this should be `selectV2()` with this PR change?

I don't think this alleviates the breaking change, it only defers the problem to a later date.

Reply:
yes, seems that by doing that we could release this early and deprecate the old method, and then remove the old implementation at the next major.

Reply:
yep, this means the breaking change in v4 will be renaming `selectV2` to `select`

this was meant to be `selectV2`, I've updated it 👍